content/documentation/frontend-layer/templating.md

```diff
@@ -8,70 +8,238 @@ aliases:
-<em>Please see <a href="https://github.com/gobuffalo/plush" target="_blank">github.com/gobuffalo/plush</a> for more details on the underlying templating package.</em>
-</p
+{{<note>}}
+This document only applies when using [https://github.com/gobuffalo/buffalo/tree/main/render](https://github.com/gobuffalo/buffalo/tree/main/render).
+Please see [github.com/gobuffalo/plush](https://github.com/gobuffalo/plush) for more details on the underlying templating package.
+{{</note>}}
+
 
 Buffalo defaults to using [plush](https://github.com/gobuffalo/plush) as its template engine.
 
-<div class="row">
-  <div class="col-md-6">
-    {{< vimeo 207200621>}}
-  </div>
-  <div class="col-md-6">
-    {{< vimeo 267643588>}}
-  </div>
-</div>
+## Introduction to Plush
+{{< vimeo 207200621>}}
+
+## Plush - Tips, Tricks and Testing
+
+{{< vimeo 267643588>}}
 
 ## General Usage
 
+Plush allows you to capture the `context` variables to use anywhere in your templates.
+
+{{<codetabs>}}
+{{<tab "actions/index.go">}}
+```go
+func myHandler(c buffalo.Context) error {
+  c.Set("name", "John Smith")
+  return c.Render(http.StatusOK, r.HTML("index.plush.html"))
+}
+```
+{{</tab>}}
+{{<tab "templates/index.plush.html">}}
 ```erb
-// templates/index.html
 <h1><%= name %></h1>
-<ul>
-  <%= for (name) in names { %>
-    <li><%= name %></li>
-  <% } %>
-</ul>
 ```
+{{</tab>}}
+{{<tab "Output">}}
+```html
+<h1>John Smith</h1>
+```
+{{</tab>}}
+{{</codetabs>}}
 
+## Plush Examples
+
+#### Conditional Statements
+
+{{<codetabs>}}
+{{<tab "IF">}}
+```erb
+<%= if (true) { %>
+  <!-- some template content -->
+<% } %>
+```
+{{</tab>}}
+{{<tab "ELSE">}}
+```erb
+<%= if (true) { %>
+  <!-- content when statement is true -->
+<% } else { %>
+  <!-- content when statement is false -->
+<% } %>
+```
+{{</tab>}}
+{{<tab "ELSE IF">}}
+```erb
+<%= if (value == 0) { %>
+  <!-- content when value is 0 -->
+<% } else if (value == 1) { %>
+  <!-- content when value is 1 -->
+<% } else { %>
+  <!-- content when value is different to 0 and 1 -->
+<% } %>
+```
+{{</tab>}}
+{{<tab "Multiple Conditions">}}
+```erb
+<%= if ((value > 0) && (value < 10)) { %>
+  <!-- some template content -->
+<% } else { %>
+  <!-- some template content -->
+<% } %>
+```
+{{</tab>}}
+{{</codetabs>}}
+
+{{<codetabs>}}
+{{<tab "actions/main.go">}}
 ```go
-// actions/index.go
-func IndexHandler(c buffalo.Context) error {
-  c.Set("name", "Mark")
+func MyHandler(c buffalo.Context) error {
+	// ...
+	c.Set("userName", "John Smith")
+	return c.Render(http.StatusOK, r.HTML("templates/index.plush.html"))
+}
+```
+{{</tab>}}
+{{<tab "templates/index.plush.html">}}
+```erb
+<%= if (userName != "") { %>
+  <span>Welcome <strong><%= userName %>!</strong></span>
+<% } else { %>
+  <span>Welcome Visitor</span>
+<% } %>
+```
+{{</tab>}}
+{{<tab "Output">}}
+```html
+<span>Welcome <strong>John Smith!</strong></span>
+```
+{{</tab>}}
+{{</codetabs>}}
+
+### Iterating
```

Comment:
I think it should be `H2` rather than `H3`. (and the followings are needed one step up too)